src/annotator/integrations/test/html-test.js

```diff
@@ -354,4 +366,66 @@ describe('HTMLIntegration', () => {
       assert.deepEqual(await integration.uri(), 'https://example.com/');
     });
   });
+
+  it('emits "uriChanged" event when URL changes after a navigation', () => {
+    const integration = createIntegration();
+    const onURIChanged = sinon.stub();
+    integration.on('uriChanged', onURIChanged);
+
+    fakeHTMLMetadata.uri.returns('https://example.com/new-url');
+    notifyNavigation();
+
+    assert.calledWith(onURIChanged, 'https://example.com/new-url');
+  });
+
+  it('emits "uriChanged" event when URL changes after a <head> change', async () => {
```

Comment:
Might wrap this and related tests in a `context('after <head> changes')` or similar?

Reply:
Currently the tests are organized as cases where "uriChanged" is emitted, followed by cases where it is not, rather than by what does/does not trigger it. It might make sense to group by trigger. I'd like to wait until additional tests are needed to make the appropriate grouping clearer.